src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -1019,6 +1019,27 @@ void CodeGen::genSSE2Intrinsic(GenTreeHWIntrinsic* node)
             break;
         }
 
+        case NI_SSE2_SetScalarVector128:
+        {
+            assert(baseType == TYP_DOUBLE);
+            assert(op2 == nullptr);
+
+            instruction ins = Compiler::insOfHWIntrinsic(intrinsicID, node->gtSIMDBaseType);
+            if (op1Reg == targetReg)
+            {
+                regNumber tmpReg = node->GetSingleTempReg();
+
+                assert(tmpReg != targetReg);
```

Comment:
Please ensure you copy over comments as well when copying code (missing `// Ensure we aren't overwriting targetReg`)

Reply:
Will fix